### PR TITLE
Some API tests

### PR DESCRIPTION
**Pull request by anissa-agahchen**

### CODE REVIEW

File: api/src/paths/project/{projectId}/attachments/list.test.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      query: mockQuery
+    });
+
+    sinon.stub(project_attachments_queries, 'getProjectAttachmentsSQL').returns(SQL`something`);
+
+    const result = listAttachments.getAttachments();
+
+    await result(sampleReq, sampleRes as any, (null as unknown) as any);
+
+    expect(actualResult).to.not.be.null;
```
Comment (sdevalapurkar) [inline api/src/paths/project/{projectId}/attachments/list.test.ts]: should be able to assert the actual value of the result, not just that it isnt null

Comment (anissa-agahchen) [reply api/src/paths/project/{projectId}/attachments/list.test.ts]: Yeah, not super happy with this test.  I seem to not be able to see the id in the actualResult and the others are undefined.

Comment (anissa-agahchen) [reply api/src/paths/project/{projectId}/attachments/list.test.ts]: should be fixed now

**Review by sdevalapurkar — APPROVED** [review]
download

**Pull request merged**